Story-Chief/frontend-test-erikahendsel
# Event type filter fix. Showing all event now.

## Description


Event types were showing the events that had multiple filters selected and not filter by type on each own. The change now is able to show all event types that user filtered on.

## Closes issue(s)


--

## Testing Instructions






1. Go to Event Types input
2. Select Sports-related event
**Result**: You see only sports-related events
3. Select a Food-related event.
**Result**: Now it should show Sport-related and Foond-related events
4.Add more filters
**Result**: Any filter you select should show in the calendar. Anything that is not selected should NOT be shown.


## Screenshots


![image](https://user-images.githubusercontent.com/38254577/219617991-7635af2e-277f-4e1b-8d76-9d10add6de8b.png)
